### PR TITLE
feature: keep prediction in iu unitll next run

### PR DESCRIPTION
**Pull request by alex-krasn**
on issue #204

### CODE REVIEW

File: src/react/components/pages/predict/predictPage.tsx [hunk 1]
```diff
@@ -838,9 +850,19 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
         this.imageMap.addFeatures(features);
     }
 
+    private saveLastPredictionResultToLocalStorage = (result, projectID) => {
```
Comment (kunzms) [inline src/react/components/pages/predict/predictPage.tsx:853]: what about only save only one project (last run) result? Don't want to consume too much local storage to exceed to local storage quota

Comment (alex-krasn) [reply src/react/components/pages/predict/predictPage.tsx:853]: Yes, It's make sense since Local Storage is pretty limited.

Comment (alex-krasn) [reply src/react/components/pages/predict/predictPage.tsx]: implemented in - d962fd0

**Comment by alex-krasn** [comment]
relocated to https://github.com/microsoft/OCR-Form-Tools/pull/285

**Pull request closed without merging**